### PR TITLE
Hello rails react

### PR DESCRIPTION
**Pull request by Gh-Reza**
## Changes:

- Created the rails and react app using Webpack
- Connected the rails and react
- Created the API end point
- Added react component

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+  </head>
```
Comment (brainconnect93) [inline app/views/layouts/application.html.erb:11]: - [ ] Great effort working on your project, I could check the API endpoint and it is working perfectly fine. But I get errors when I check the random greetings rendered on the UI. Below is a video demonstration of the project behavior at the moment.

Video 👇 


https://github.com/Gh-Reza/hello-rails-react/assets/78634841/5103a9a4-a12a-4898-9992-2705e9f1ba98

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Gh-Reza, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- "Hello world" project with `react-rails` gem ✔️
- Implement the `API` in the backend correctly ✔️
- Implement the front end correctly ✔️ 
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Gh-Reza) [reply app/views/layouts/application.html.erb:11]: Hey @brainconnect93, thank you for reviewing my code.
It is working on my computer and my friend's computer. Don't know why it is not working on your computer

image
image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Reza,  

**Set up a 'Hello world' project as one app with Webpack** project is complete! 🏁Congratulations!  🎉 💯 🌟 

## Highlights 🎯 
🎲 Connection between Rails back-end and React front-end properly established in rails app!
🎲 Random greetings generating correctly!
🎲 ReadMe looks great!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**